### ISSUE TITLE
tiling error on world dataset

### ISSUE THREAD

**Issue by jrjdavidson** [Melown/vts-backend#27]
Got this error while running mapproxy-setup-resource on a world DEM dataset. It's strange because I had already processed this specific dataset, had just copied the raw data over and tried to process it again. ( i thought it would be faster than trying to copy gigs of data).

```
2021-09-03 20:05:41 I1 (tiling) [46738(tile:2-1-3)]: Coordinate system transformation (PROJCS["unknown",GEOGCS["unknown",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]]],PROJECTION["Polar_Stereographic"],PARAMETER["latitude_of_origin",90],PARAMETER["central_meridian",0],PARAMETER["scale_factor",0.994],PARAMETER["false_easting",2000000],PARAMETER["false_northing",2000000],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AXIS["Easting",SOUTH],AXIS["Northing",SOUTH]] -> GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AXIS["Latitude",NORTH],AXIS["Longitude",EAST],AUTHORITY["EPSG","4326"]]). {csconvertor.cpp:CsConvertor():446}
2021-09-03 20:05:41 I1 (tiling) [46738(tile:2-1-3)]: Setting SOURCE_EXTRA=438 and SAMPLE_GRID=YES. {geodataset.cpp:sourceExtra():862}
2021-09-03 20:05:41 I1 (tiling) [46738(tile:2-1-3)]: Memory requirements at overview 4: 6748624 bytes. {geodataset.cpp:overviewByMemoryReqs():1085}
2021-09-03 20:05:41 I3 (tiling) [46738(tile:2-3-0)]: Processed tile 2-3-0 (extents: 2000000.000000,2000000.000000:3721400.000000,3721400.000000, srs: steres-wgs84) [watertight]. {tiling.cpp:process():339}
2021-09-03 20:05:41 I2 (tiling) [46738(tile:3-0-2)]: Processing tile 3-0-2. {tiling.cpp:process():242}
2021-09-03 20:05:41 I1 (tiling) [46738(tile:3-0-2)]: Overview selection based on dst/src scale of 0.00941. {geodataset.cpp:overviewByScale():1029}
2021-09-03 20:05:41 I1 (tiling) [46738(tile:3-0-2)]: Most efficient lossless source is overview #5. {geodataset.cpp:chooseOverview():1141}
2021-09-03 20:05:41 E1 (tiling) [46738(tile:2-3-1)]: Cannot convert point [2](1986551.562500,2013448.437500) between coordinate systems: <>. {csconvertor.cpp:convert():133}
terminate called after throwing an instance of 'geo::ProjectionError'
  what():  Cannot convert point [2](1986551.562500,2013448.437500) between coordinate systems: <>. @{csconvertor.cpp:convert():133}
/usr/bin/mapproxy-setup-resource: line 8: 46738 Aborted                 (core dumped) /opt/vts/bin/mapproxy-setup-resource "$@" -f "${CONFIG}"

```

**Comment by vaclavblazek** [comment]
Pls, could you grep the log for `tile:2-3-1`? There is too little info about what is done with this tile at the end of the log as it runs in parallel.

**Comment by jrjdavidson** [comment]
Do logs get saved anywhere? I’ve already deleted the dataset to try another
method..

**Comment by vaclavblazek** [comment]
No, you need to explicitely specify the log file via `--log.file PATH`.
I'm affraid I can't investigate what went wrong without log and/or data.

**Comment by jrjdavidson** [comment]
Goes beyond 180?
https://www.dropbox.com/s/xhxjq7gfjfc8zcy/log.txt?dl=0

**Comment by vaclavblazek** [comment]
It's strange, the error message from GDAL is empty (quoted by `<>`).
How large is the dataset? Would it be possible to share it?
Also, Ubuntu/Focal?

**Comment by jrjdavidson** [comment]
its a 800 Mb file, I'll upload it somewhere to share - it's the de ferranti 15 arc s world DEM all merged into one tif
it's strange because i had already created a resource from that exact file before on our local server (ubuntu 18). Now I'm working on ubuntu 20 on azure.

**Comment by vaclavblazek** [comment]
Thank you. 

We've found some problems in our internal SW on `focal` with stock GDAL (3.0.4) so we had to switch to newer GDAL (3.2.1) from `ubuntugis` repo. Maybe this might be some related problem. I'll try it to run it against both versions once I have the data.

**Comment by jrjdavidson** [comment]
Sent to your @melowntech address

**Comment by jrjdavidson** [comment]
Assuming this is solved by #28
